syntax = "proto3";

import "tweetMessages.proto";

service RandomLang {
    rpc ReturnRandomLang (RandomLangRequest) returns (RandomResponse);
}
